include/reactor-uc/startup_coordinator.h

```diff
@@ -4,17 +4,26 @@
 #include "reactor-uc/error.h"
 #include "reactor-uc/tag.h"
 #include "reactor-uc/event.h"
+#include "reactor-uc/reactor.h"
 #include "proto/message.pb.h"
 
 typedef struct StartupCoordinator StartupCoordinator;
 typedef struct Environment Environment;
+typedef enum JoiningPolicy JoiningPolicy;
+typedef struct TimerConfig TimerConfig;
+
+enum JoiningPolicy { JOIN_IMMEDIATELY = 0, JOIN_ALIGNED_WITH_SHORT_TIMER = 1, JOIN_AT_HYPER_PERIOD = 2 };
 
 /** Represents the state of a neighbor. */
 typedef struct {
+  bool core_federate;                   // Whether this federate can be not available
```

Comment:
The name of this variable and the comment contradict each other. Per the comment, if it is true, then it can be not available (is transient). Per the variable name, it's the opposite.

Reply:
This property is relevant for how many responses to my `StartTimeRequest` I can expect from my neighbors. 

https://github.com/lf-lang/reactor-uc/pull/251/files#diff-73d218f09f4226225ac8339e6497116d217cd3ebd26cbe070501e92126de3883R418

Currently I wait until I have a response from all my neighboring core federates.